arkouda/pdarraycreation.py

```diff
@@ -469,15 +481,16 @@ def arange(*args) -> pdarray:
         raise TypeError("start,stop,stride must be type int {} {} {}".\
                                     format(start,stop,stride))
 
+@typechecked
 def linspace(start : int, stop : int, length : int) -> pdarray:
```

Comment:
This doesn't seem correct, `start` and `stop` can be `float`

Reply:
@mhmerrill Ugh, whoops, fixing now.

Reply:
Fixed